### PR TITLE
Preparing info for ArtifactHub.io

### PR DESCRIPTION
**Pull request by didierofrivia**
In order to be able to publish our Helm Charts in ArtifactHub.io and display valuable information in their repo site, we should provide some metadata described in https://artifacthub.io/docs/topics/annotations/helm/

### CODE REVIEW

File: charts/kuadrant-operator/Chart.yaml [hunk 1]
```diff
@@ -44,3 +44,102 @@ maintainers:
...
+annotations:
+  artifacthub.io/category: networking
+  artifacthub.io/crds: |
+    - kind: Kuadrant
+      version: v1beta1
+      name: kuadrants.kuadrant.io
+      displayName: Kuadrant
+      description: Kuadrant configures installations of Kuadrant Service Protection components.
+    - kind: AuthPolicy
+      version: v1beta1
```
Comment (eguzki) [inline charts/kuadrant-operator/Chart.yaml:56]: `v1beta3` ?

File: charts/kuadrant-operator/Chart.yaml [hunk 2]
```diff
@@ -44,3 +44,102 @@ maintainers:
...
+      name: authpolicies.kuadrant.io
+      displayName: AuthPolicy
+      description: AuthPolicy enables authentication and authorization for service workloads in a Gateway API network.
+    - kind: RateLimitPolicy
+      version: v1beta1
```
Comment (eguzki) [inline charts/kuadrant-operator/Chart.yaml:61]: `v1beta3` ?

File: charts/kuadrant-operator/Chart.yaml [hunk 3]
```diff
@@ -44,3 +44,102 @@ maintainers:
...
+      name: ratelimitpolicies.kuadrant.io
+      displayName: RateLimitPolicy
+      description: RateLimitPolicy enables rate limiting for service workloads in a Gateway API network.
+    - kind: DNSPolicy
+      version: v1beta1
```
Comment (eguzki) [inline charts/kuadrant-operator/Chart.yaml:66]: Check this out https://github.com/Kuadrant/kuadrant-operator/pull/973

File: charts/kuadrant-operator/Chart.yaml [hunk 4]
```diff
@@ -44,3 +44,102 @@ maintainers:
...
+      name: dnspolicies.kuadrant.io
+      displayName: DNSPolicy
+      description: DNSPolicy is the Schema for the dnspolicies API.
+    - kind: TLSPolicy
+      version: v1beta1
```
Comment (eguzki) [inline charts/kuadrant-operator/Chart.yaml:71]: Check this out https://github.com/Kuadrant/kuadrant-operator/pull/973

**Review by eguzki — CHANGES_REQUESTED** [review]

**Comment by eguzki** [comment]
we are moving to `v1` for all policies. Probably good to hold on this PR a bit. I would assume all of them are `v1`

File: charts/kuadrant-operator/Chart.yaml [hunk 5]
```diff
@@ -44,3 +44,102 @@ maintainers:
     name: Jim Fitzpatrick
   - email: chfan@redhat.com
     name: Kevin Chi Fan
+annotations:
+  artifacthub.io/category: networking
+  artifacthub.io/crds: |
+    - kind: Kuadrant
+      version: v1
```
Comment (didierofrivia) [inline charts/kuadrant-operator/Chart.yaml:51]: @maleck13 should Kuadrant CRD be promoted to v1 too?

Comment (maleck13) [reply charts/kuadrant-operator/Chart.yaml]: 👎

**Review by eguzki — APPROVED** [review]

**Pull request merged**